powered-by-art-blocks-pbab-onboarding/pbab-101/what is powered by art blocks

```diff
@@ -0,0 +1,49 @@
+# What is Powered by Art Blocks?
+Powered by Art Blocks (PBAB) is a custom branded solution from Art Blocks.  It allows the generative NFT minting technology used by artists at Art Blocks to be integrated with third-party sites. 
+
+PBAB allows partners to release generative outputs using our existing smart contracts and rendering infrastructure resulting in turnkey and branded generative projects. PBAB partners own their smart contracts and can use them to release as many projects as they like. 
+
+We currently partner with organizations from every sector that are interested in launching generative projects.
+
+For more information on Powered by Art Blocks partnerships please contact: info@artblocks.io
+
+# What is generative art?
+
+Generative art is about developing systems that define rules for creating art. By introducing randomness to those systems, core concepts are expressed through unique outputs. In a contemporary sense, this means writing computer algorithms to define the system and introduce randomness, which allows for conceptual exploration and rapid iteration. 
+
+Creative coders write scripts with specific parameters that introduce and explore features, which generate the final outputs in a collection. With Art Blocks’ generative minting technology, collectors participate in the creation of the art. No one knows exactly what a piece will look like before it's minted, not even the artist. EachNFT is generated at the time of purchase using the buyer’s unique transaction hash to create a ‘1 of 1 of X’, which adds an extra layer of magic as the creator and collector watch a project come alive. 
+
+The interesting part of modern generative art is that it involves working in series - often a large series. So instead of pursuing a single compelling work of art, a generative artist creates an algorithm capable of tens, hundreds, or thousands of compelling works of art. Which, when taken as a whole, expresses the range of possibilities contained in a single algorithm.
+
+# What are NFTs? 
+Nonfungible tokens (NFT) are unique digital assets stored on blockchain technology that can represent any digital or physical asset. 
+
+# On-chain vs. off-chain
+Powered by Art Blocks (PBAB) enables creators to immutably store their generative NFT directly on the Ethereum blockchain (on-chain) or reference an external library or asset (off-chain)
```

Comment:
I think we should specify here that we allow referencing an external asset that is off-chain but referenced via decentralized storage solutions using IPFS.